src/components/Toolbar/Toolbar.scss

```diff
@@ -85,3 +85,41 @@ body:global(.expanded) {
   right: 14px;
   top: 4px;
 }
+
+.stats {
+  position: absolute;
+  right: 20px;
+  bottom: 10px;
+  background: $blue;
+  border-radius: 3px;
+  padding: 2px 4px;
+  font-size: 14px;
```

Comment:
I've changed the font size from 16px to 14px to align with the other font sizes in the toolbar. Not sure but I may have messed up the layout of the drop down, could you please take a look? Then this is ready to merge.

Reply:
Updated the styles. Now it would look like this.

 
<img width="124" alt="Screenshot 2024-02-26 at 2 08 02 PM" src="https://github.com/parse-community/parse-dashboard/assets/49753983/99400c64-be5a-4bdd-8e82-30c219a38152">